XenAdmin/Wizards/PatchingWizard/PatchingWizard_PrecheckPage.cs

```diff
@@ -601,12 +612,9 @@ private void buttonResolveAll_Click(object sender, EventArgs e)
                         actions.Add(action);
                 }
             }
-            foreach (var asyncAction in actions)
-            {
-                _progressDialog = new ActionProgressDialog(asyncAction, ProgressBarStyle.Blocks);
-                _progressDialog.ShowDialog(this);
-                Program.Invoke(Program.MainWindow, RefreshRechecks);
-            }
+            var multipleAction = new CrossConnectionParallelAction(Messages.PATCHINGWIZARD_PRECHECKPAGE_RESOLVING_ALL, "", "", actions);

```

Comment:
It could be worth writing a comment here explaining why the other two strings aren't needed.